gradle/libs.versions.toml

```diff
@@ -16,6 +16,8 @@ managed-protobuf = '0.8.18'
 managed-zipkin-reporter = '2.16.3'
 managed-opentelemetry-contrib-aws-xray = '1.15.0'
 managed-opentelemetry-gcp-trace = '0.22.0-alpha'
+managed-aws-java-sdk-v2 = '2.17.196'
```

Comment:
I don't think this should be managed here, it should be managed by the Micronaut AWS module

Reply:
resolved via https://github.com/micronaut-projects/micronaut-tracing/pull/110/commits/f264f9fbdf295f8bb64810205721769a86253554